myndla-api/src/main/scala/no/ndla/myndlaapi/controller/ArenaController.scala

```diff
@@ -0,0 +1,451 @@
+/*
+ * Part of NDLA myndla-api
+ * Copyright (C) 2023 NDLA
+ *
+ * See LICENSE
+ */
+
+package no.ndla.myndlaapi.controller
+
+import io.circe.generic.auto._
+import no.ndla.myndla.MyNDLAAuthHelpers
+import no.ndla.myndla.model.api.ArenaOwner
+import no.ndla.myndla.service.UserService
+import no.ndla.myndlaapi.Eff
+import no.ndla.myndlaapi.model.arena.api.{
+  Category,
+  CategoryWithTopics,
+  NewCategory,
+  NewFlag,
+  NewPost,
+  NewTopic,
+  PaginatedNewPostNotifications,
+  PaginatedPosts,
+  PaginatedTopics,
+  Post,
+  Topic,
+  TopicWithPosts
+}
+import no.ndla.myndlaapi.service.ArenaReadService
+import no.ndla.network.clients.FeideApiClient
+import no.ndla.network.tapir.NoNullJsonPrinter.jsonBody
+import no.ndla.network.tapir.TapirErrors.errorOutputsFor
+import no.ndla.network.tapir.{Service, TapirErrorHelpers}
+import sttp.model.StatusCode
+import sttp.tapir.generic.auto._
+import sttp.tapir.server.ServerEndpoint
+import sttp.tapir._
+
+trait ArenaController {
+  this: ErrorHelpers
+    with TapirErrorHelpers
+    with MyNDLAAuthHelpers
+    with ArenaReadService
+    with FeideApiClient
+    with UserService =>
+  val arenaController: ArenaController
+
+  class ArenaController extends Service[Eff] {
+    import MyNDLAAuthHelpers.authlessEndpointFeideExtension
+    override val serviceName: String                   = "Arena"
+    override protected val prefix: EndpointInput[Unit] = "myndla-api" / "v1" / "arena"
```

Comment:
Meir i henhold til andre apis
```suggestion
    override val serviceName: String                   = "arena"
    override protected val prefix: EndpointInput[Unit] = "myndla-api" / "v1" / serviceName
```